pkg/ride/compiler/ast_parser.go

```diff
@@ -136,7 +136,7 @@ func (p *astParser) loadBuildInVarsToStackByVersion() {
 			Type: txType,
 		})
 	}
-	if p.tree.LibVersion >= ast.LibV4 && p.tree.LibVersion <= ast.LibV7 {
+	if p.tree.LibVersion >= ast.LibV4 && p.tree.LibVersion <= ast.LibV8 {
```

Comment:
Maybe use `CurrentMaxLibraryVersion` instead of `ast.LibV8`?

Reply:
It fixed in #1140.